Automattic/wp-calypso
# Gutenboarding: add PlansGrid accordion

#### Changes proposed in this Pull Request
* Re-add plans accordion in Gutenboarding and Site setup. **Note: PlansGrid accordion was first introduced in #44299**
* Ship Feature picker and Plans Grid accordion to Gutenboarding "stable" (still behind `gutenboarding/feature-picker` flag in production environment).
* Update Gutenboarding e2e test.

#### Testing instructions
* Go to [/new](https://hash-88877d0524e87950b8e6138c974bc58e78661c17.calypso.live/new)
* Before reaching Plans step in the linear flow, you should land on Feature selection step.
* After skipping or confirming feature selection, Plans step should display as an accordion.
* After creating the site, apply D48381-code and sandbox the new site.
* When in editor Site setup flow should include PlansGrid accordion.
